diff --git a/tools/cmake/gen_binary.cmake b/tools/cmake/gen_binary.cmake
index e8a100e..f61ca6e 100644
--- a/tools/cmake/gen_binary.cmake
+++ b/tools/cmake/gen_binary.cmake
@@ -28,7 +28,6 @@ endif()
 set(cp_assets_cmd COMMAND python ${SDK_PATH}/tools/cmake/copy_assets.py ${PROJECT_PATH} ${PROJECT_DIST_DIR}/${PROJECT_ID}_${build_type})
 
 add_custom_command(TARGET ${PROJECT_ID} POST_BUILD
-    ${strip_cmd}
     COMMAND mkdir -p ${CMAKE_BINARY_DIR}/dl_lib
     ${cp_command}
     ${cp_dist_cmd}
